ros/src/waypoint_updater/waypnt_updater.py

```diff
@@ -510,24 +518,23 @@ def get_stopping_time(self, start, end):
                               .format(counter))
                 optimized = True
 
-        rospy.logdebug("found jerk of {:3.2f} with a_dist of {:3.2f}"
-                       " and acc of {:3.2f} with a time of {:3.2f}"
-                       .format(jerk, a_dist, acc ,T))
-            
+        rospy.logdebug("found initial jerk of {:3.2f}m/s^3 and final acc of {:3.3f}m/s^2"
+                       " using a_dist of {:3.2f}m and time of {:3.2f}s"
+                       .format(jerk, acc, a_dist, T))
 
         duration = rospy.get_time() - timer_start
-        rospy.loginfo("Shortest Distance to decelerate with max_jerk={:3.3f}"
-                      "from v={:3.3f}, a={:3.3f} to v={:3.3f} in dist {:3.3f}"
-                      " m in time {:3.3f} s - took {:3.4f} s to calc"
-                      .format(jerk, start[1], start[2], end[1],
+        rospy.loginfo("Shortest Distance to decelerate with max_jerk={:3.3f}m/s^3 "
+                      "from v={:3.3f}m/s, a={:3.3f}m/s^2 to v={:3.3f}m/s a={:3.3f}m/s^2 in dist {:3.3f}m"
+                      " in time {:3.3f}s - Took {:3.4f}s to calc."
+                      .format(jerk, start[1], start[2], end[1], end[2],
                               a_dist, T, duration))
         return T
 
     def setup_stop_jmt(self, ptr_id, a_dist):
         # this is set up to stop the car in a desired distance
 
         curpt = self.waypoints[ptr_id]
-        target_velocity = 1.10 * self.min_moving_velocity # was 0.0, move to just above creep speed
+        target_velocity = 1.25 * self.min_moving_velocity # was 1.1, move to just above creep speed
```

Comment:
I found that the car did not reach the target zone if this was set to a smaller value (1.15 * min_moving_velocity), and waypoint updater would have car in slowdown , but car would be stopped.